File: 2021/May/Day 12/goal-parser.py
# Sourc : https://leetcode.com/problems/goal-parser-interpretation/submissions/
class Solution(object):
    def interpret(self, command):
        """
        :type command: str
        :rtype: str
        """
        string =""
        for i in range(len(command)):

            if(command[i]=='G'):
                string+="G";
            elif(command[i:i+2]=="()"):
                string+="o"
            elif(command[i:i+4]=="(al)"):
                string+="al"
        return string

File: 2023/February/Day 9/goalParser.cpp
// Source : https://leetcode.com/problems/goal-parser-interpretation/submissions/
class Solution {
public:
    string interpret(string command) {
        string out="";
        for(int i=0;i<size(command);i++)
        {
            if(command[i]=='(')
            {
                if(command[i+1]==')')
                {
                    out+="o";
                    
                }
                else
                {
                    out+="al";
                }
            }
            else if (command[i]=='G')
            {
                out+="G";
            }
            
            
        }
        return out;
    }
};